tools/flowc/backends/common.flow

```diff
@@ -179,6 +179,8 @@ export {
 		javaLibrary : Set<string>,
 		includes : [string],
 		debug : bool,
+		useLambdas : bool,        // Use lambdas instead of inner classes for closures
+		staticRuntime : bool,     // All methods in runtime are static
```

Comment:
Why do we need this? How can it do work?

Reply:
I mean native methods cannot be non-static now, right?

Reply:
1) No, I'll remove these config parameters.
2) yes, in static runtime all native methods must be static